python/ray/services.py

```diff
@@ -185,6 +185,9 @@ def address_to_ip(address):
     """
     address_parts = address.split(":")
     ip_address = socket.gethostbyname(address_parts[0])
+    # Make sure localhost isn't resolved to the loopback ip
+    if ip_address == "127.0.0.1":
+        ip_address = get_node_ip_address()
```

Comment:
As discussed in #1534, this command fails without internet access, so we should probably include a fix for that in this PR as well. I can push something.